Library/Formula/psqlodbc.rb

```diff
@@ -1,9 +1,23 @@
-require "formula"
-
 class Psqlodbc < Formula
-  homepage "http://www.postgresql.org/"
-  url "http://ftp.postgresql.org/pub/odbc/versions/src/psqlodbc-09.03.0300.tar.gz"
-  sha1 "0f41b4678b513aa70f14b03803af92a7abf1e179"
+  homepage "https://odbc.postgresql.org"
+
+  stable do
+    url "http://ftp.postgresql.org/pub/odbc/versions/src/psqlodbc-09.03.0400.tar.gz"
+    sha256 "de77dfa89dba0a159afc57b2e312ca6e9075dd92b761c7cc700c0450ba02b56b"
+
+    # fixes error "conflicting types for '__builtin___strlcat_chk'"
```

Comment:
Can you link to the issue where you got this patch/has it been reported upstream?